l10n_it_pos_fatturapa/static/src/js/models.js

```diff
@@ -0,0 +1,12 @@
+odoo.define('l10n_it_pos_fatturapa.fatturapa_field', function (require) {
+    "use strict";
+
+  var pos_models = require('point_of_sale.models');
+  pos_models.load_fields("res.partner", "electronic_invoice_subjected");
+  pos_models.load_fields("res.partner", "eori_code");
+  pos_models.load_fields("res.partner", "codice_destinatario");
+  pos_models.load_fields("res.partner", "pec_destinatario");
+  pos_models.load_fields("res.partner", "pa_partner_code");
```

Comment:
puoi caricare tutti i field in un'unica chiamata:

`pos_models.load_fields("res.partner", ["codice_destinatario", "pec_destinatario", ...]);
`

Reply:
@GSLabIt Fatto!